alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/alerts/AlertMover.kt

```diff
@@ -94,15 +93,15 @@ class AlertMover(
 
     private fun onSearchResponse(response: SearchResponse) {
         // If no alerts are found, simply return
-        if (response.hits.totalHits == 0L) return
+        if (response.hits.totalHits.value == 0L) return
         val indexRequests = response.hits.map { hit ->
-            IndexRequest(AlertIndices.HISTORY_WRITE_INDEX, AlertIndices.MAPPING_TYPE)
+            IndexRequest(AlertIndices.HISTORY_WRITE_INDEX)
                     .routing(monitorId)
                     .source(Alert.parse(alertContentParser(hit.sourceRef), hit.id, hit.version)
                             .copy(state = Alert.State.DELETED)
                             .toXContent(XContentFactory.jsonBuilder(), ToXContent.EMPTY_PARAMS))
-                    .version(hit.version)
-                    .versionType(VersionType.EXTERNAL_GTE)
+                    .setIfSeqNo(hit.seqNo)
+                    .setIfPrimaryTerm(hit.primaryTerm)
```

Comment:
Same as above in order to use `seqNo` you would need to enable in search via `seq_no_primary_term` to `true`